core/src/main/scala/filodb.core/binaryrecord2/RecordBuilder.scala

```diff
@@ -540,7 +540,8 @@ class RecordBuilder(memFactory: MemFactory,
       val oldOffset = curRecordOffset
       if (reuseOneContainer) resetContainerPointers() else newContainer()
       logger.trace(s"Moving $recordNumBytes bytes from end of old container to new container")
-      require((containerSize - ContainerHeaderLen) > (recordNumBytes + numBytes), "Record too big for container")
+      if((containerSize - ContainerHeaderLen) <= (recordNumBytes + numBytes))
```

Comment:
Update the readme as technically this is a breaking change as we change the exception type from ``IllegalArgumentException``thrown by ``require``  to ``RecordOutOfContainerCapacityException`` which extends  ``RuntimeException``